### ISSUE TITLE
Fixing OAuth documentation on h.readthedocs.io

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/support#97]
Problems:

1) Internally, there are different opinions about when we use OAuth vs SSO vs Third-party integration. We should pick one set of terms for each process and use them consistently in conversations within and without the company.

For requests where we provide an OAuth token, users create Hypothesis accounts (authority = hypothes.is), we should call that "OAuth".

For requests where we provide a third-party integration (authority != hypothes.is) we should call that "third-party", or "third-party/SSO".

2) Our documentation doesn't discriminate between OAuth and 3rd-party/SSO. I've added some annotations displaying this confusion [in this group](https://hypothes.is/groups/EYmX5oPm/read-the-docs-notes) here:

- https://hyp.is/X_-_nmMREeq9M1eXnTKHbw/h.readthedocs.io/en/latest/api/authorization/
- https://hyp.is/X_-_nmMREeq9M1eXnTKHbw/h.readthedocs.io/en/latest/api/authorization/

Proposed solutions:

1. Standardize language internally and on h.readthedocs.io, differentiating between OAuth and 3rd-party/SSO.
2. Correct instances where our documentation either references the wrong thing, or comes to an incorrect conclusion.

**Comment by mkdir-washington-edu** [comment]
I've tried to make a start of this, and will continue to update with feedback.

I also wanted to note @judell 's suggestion, which was to get rid of the "OAuth" documentation and just have folks talk to us. I would still like to clarify what we can do in each instance, since I feel like my lack of expertise has only led me to be more confused by what we can and can not do.

**Comment by judell** [comment]
>  to get rid of the "OAuth" documentation

No, that's fine. What almost all devs need to know about OAuth is nicely described at  https://h.readthedocs.io/en/latest/api/authorization/. There I learn how to make an app that uses the normal H client/server login machinery. (You saw an example of this recently when, in addition to the Google auth at labs.hypothes.is, we also did H auth.)

Very few devs need to know about https://h.readthedocs.io/en/latest/publishers/authorization-grant-tokens/, and we don't really want to promote it. Finding that info at the prominent link https://h.readthedocs.io/en/latest/publishers/ confuses them. I think removing that section from the doc solves that unnecessary confusion.

Similarly, that prominent link's title -- Advice for publishers -- leads publishers astray. What almost all publishers need to know about integrating H starts here: https://web.hypothes.is/publishing/. The only users  of the authorization grant token feature at https://h.readthedocs.io/en/latest/publishers/ are elife and our own LMS devs, and again we don't want to promote it widely.  I think removing that same section from the doc avoids unnecessarily confusing publishers.

**Comment by xolotl** [comment]
Happy to help. I didn't get a clear picture of an action to take reading this issue.

**Comment by judell** [comment]
Either remove https://h.readthedocs.io/en/latest/publishers/ from readthedocs, which doesn't involve you.

Or retain Advice for Publishers there, but redirect to https://web.hypothes.is/publishing/. You might have an opinion on that.

**Comment by xolotl** [comment]
Got it. Yes, I always think the website at https://web.hypothes.is/publishing/ will make more sense as the first stop for publishers than anything in RTD. RTD should focus on technical documentation.
